modules/nf-core/chewbbaca/createschema/main.nf

```diff
@@ -0,0 +1,63 @@
+process CHEWBBACA_CREATESCHEMA {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::chewbbaca=3.3.1"
```

Comment:
```suggestion
        conda "${moduleDir}/environment.yml"
```

& add `environment.yml`...see any other module for example

Reply:
resolved with https://github.com/nf-core/modules/pull/4512/commits/ecbaa6347c8ec4c6e64d6ae8a861371ae75276f1